### PR TITLE
change generateToast method place on layer buttons

### PR DESCRIPTION
**Pull request by AntoinouM**

### CODE REVIEW

File: components/EditionMenu.vue [hunk 1]
```diff
@@ -26,6 +26,17 @@
...
     }
+
+    function sendUp() {
+        emits('frontEvent');
+        generateToast('info', 'Element moved forward.');
+    }
+
+    function sendDown() {
+        emits('backEvent');
+        generateToast('info', 'Element moved backward.');
```
Comment (Annacc211018) [inline components/EditionMenu.vue:37]: I changed the text again as requested in my Pr. it is now:

Element moved forward one step.
Element moved backward one step.

to make it crystal clear to the user ^^

**Review by Annacc211018 — COMMENTED** [review]
That solved it :) Left you a comment about the new toast notification.

File: components/EditionMenu.vue (hunk 1 above)
Comment (AntoinouM) [reply components/EditionMenu.vue:37]: ok

File: components/EditionMenu.vue [hunk 2]
```diff
@@ -35,19 +46,12 @@
 <template>
     <div>
         <div class="icon-container">
-            <div class="edit-icon icon" @click="$emit('deleteEvent')" title="Delete">delete</div>
```
Comment (Annacc211018) [inline components/EditionMenu.vue:38]: it seems like you removed the titles at merge

**Review by Annacc211018 — CHANGES_REQUESTED** [review]

**Pull request merged**